.github/workflows/test_coverage.yml

```diff
@@ -59,10 +70,33 @@ jobs:
           python -m pip uninstall -y pymavlink
           git submodule update --init --recursive
           (cd modules/mavlink/pymavlink && DISABLE_MAVNATIVE=True MDEF="$PWD/../message_definitions" python -m pip install --user .)
-          Tools/scripts/run_coverage.py -f
+          if [[ ${{ matrix.config }} = "coverage" ]]; then
+            Tools/scripts/run_coverage.py -f
+          else
+            Tools/scripts/run_coverage.py -i
```

Comment:
Instructions say this has to be done after building binaries....  there's a second identical line that conforms to that.

Reply:
yep, I need to rework the script.
The thing is that we are checking that linux/tests and sitl/bin are present and with the right flags, but sitl_periph_gps ins't in the list. 
I should make the script more modulare for init.
So here I do the init twice. The first one build the binaries and init coverage. Then we build sitl_periph_gps binaries and launch again the init. It will then skip the build part and just redo the coverage init